### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by kayode-adechinan** [dmolony/dm3270#18]
Hi. Please, is it possible to replace all the runnable/threads parts with JavaFx tasks ?

**Comment by dmolony** [comment]
Hi

I think I mentioned that I do not support the sofware any longer. However, it is open source, so you are free to make any changes you like.

**Comment by kayode-adechinan** [comment]
Yes, i am about to support it myself, i just want to know if it can be
possible to do that, before starting any modification.
